backend/src/router.go

```diff
@@ -349,3 +351,48 @@ func getProfile(c *gin.Context) {
 		"user": users[0],
 	})
 }
+
+
+//not all of this is needed to make an update, at least according to the guide. We shall see.
+type ProfileUpdateData struct { 
+	ID        uint   `json:"-"`
+	FirstName string `json:"firstname"`
+	LastName  string `json:"lastname"`
+	Email     string `json:"email"`
+	Phone     string `json:"phone"`
+}
+
+// ADD DESCRIPTION
```

Comment:
The route documentation should be done in this commit, that way Chris knows the intended schema for the route so he can use it on the frontend.

Reply:
Done, although it's in progress due to adding tutors